chromadb/test/property/test_cross_version_persist.py

```diff
@@ -272,7 +272,15 @@ def persist_generated_data_with_old_version(
 
 # Since we can't pickle the embedding function, we always generate record sets with embeddings
 collection_st: st.SearchStrategy[strategies.Collection] = st.shared(
-    strategies.collections(with_hnsw_params=True, has_embeddings=True), key="coll"
+    strategies.collections(
+        with_hnsw_params=True,
+        has_embeddings=True,
+        with_persistent_hnsw_params=True,
+        # By default, these are set to 2000, which makes it unlikely that index mutations will ever be fully flushed
+        max_hnsw_sync_threshold=10,
+        max_hnsw_batch_size=10,
```

Comment:
without reducing the sync threshold, this test will almost never fully flush to disk, which makes the correctness guarantee weaker

without this change, the pickled metadata file was never loaded